src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -405,7 +405,7 @@ def setup_microbenchmark(workitem_directory, arch):
         # have not published yet. As a result, we hit errors of "dotnet restore". As a workaround, hard code the
         # working version until we move to ".NET 8" in the script.
         run_command(
-            get_python_name() + [dotnet_install_script, "install", "--dotnet-versions", "8.0.100-alpha.1.22558.2", "--architecture", arch, "--install-dir",
+            get_python_name() + [dotnet_install_script, "install", "-channels", "8.0-preview", "--architecture", arch, "--install-dir",
```

Comment:
Shouldn't this have two dashes?
```suggestion
            get_python_name() + [dotnet_install_script, "install", "--channels", "8.0-preview", "--architecture", arch, "--install-dir",
```
?

Reply:
Ah yes, I pushed the fixed commit to the internal site that led to successful spmi collection, but forgot to push it here.